libbeat/docs/shared-template-load.asciidoc

```diff
@@ -123,6 +143,37 @@ match the pattern.
 
 =====
 
+
+[[load-template-manually-alternate]]
+==== Load the template manually (alternate method)
+
+If the host running {beatname_uc} does not have direct connectivity to
+Elasticsearch, you can export the index template to a file and then install the
+template manually.
+
+ifdef::allplatforms[]
+
+*deb, rpm, and mac:*
+
+["source","sh",subs="attributes"]
+----
+./{beatname_lc} export template > {beatname_lc}.template.json
```

Comment:
For Winlogbeat I think it makes sense to show the `curl` command because you might export it on Windows and then import it on a *nix machine. This does mess up the conditional logic because the export command needs to be `.\{beatname_lc}.exe`.

To solve this I recommend breaking apart the export and import steps. Keep the conditional logic around the export commands. And always show how to import it using both Windows (`Invoke-RestMethod` ) and nix (`curl`).